### PR TITLE
JBPM-7763 - jBPM Executor Service is not registered in ServiceRegistry

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java [hunk 1]
```diff
@@ -152,6 +153,7 @@ public void init() {
     		try {
 		        executor.init();
 		        this.executorStarted = true;
+		        ServiceRegistry.get().register(org.kie.api.executor.ExecutorService.class.getSimpleName(), this);
```
Comment (MarianMacik) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java]: Why not to use newly added ServiceRegistry.EXECUTOR_SERVICE also here?

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java [hunk 2]
```diff
@@ -163,7 +165,8 @@ public void destroy() {
     	if (executorStarted) {
     		ExecutorServiceFactory.resetExecutorService(this);
 	    	this.executorStarted = false;
-	        executor.destroy();	        
+	        executor.destroy();	
+	        ServiceRegistry.get().remove(org.kie.api.executor.ExecutorService.class.getSimpleName());
```
Comment (MarianMacik) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java]: Same as above.

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java (hunk 1 above)
Comment (mswiderski) [reply jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java]: replaced

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java (hunk 2 above)
Comment (mswiderski) [reply jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorServiceImpl.java]: replaced

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**